packages/shared/src/lib/ios.ts

```diff
@@ -0,0 +1,30 @@
+import { isIOSNative } from './func';
+
+export enum WebKitMessageHandlers {
+  IAPProductList = 'iap-product-list',
+  IAPSubscriptionManage = 'iap-subscription-manage',
+  IAPSubscriptionRequest = 'iap-subscription-request',
+  IAPSetAppAccountToken = 'iap-set-app-account-token',
+}
+
+export const messageHandlerExists = (handler: WebKitMessageHandlers): boolean =>
+  isIOSNative() && globalThis.webkit?.messageHandlers?.[handler];
+
+export const sendMessage = (
+  handler: WebKitMessageHandlers,
+  payload: unknown,
+): void => {
+  if (!isIOSNative()) {
+    // eslint-disable-next-line no-console
+    console.error('sendMessage is only available on iOS');
```

Comment:
I would rather throw in this kind of cases and then we get global error in our events so we lnow to fix it, like this it can just silently fail for weeks